addons/GearSwap/gearswap.lua

```diff
@@ -285,15 +285,15 @@ function incoming_chunk(id,data,modified,injected,blocked)
             refresh_globals()
             next_packet_events.globals_update = data:unpack('H',3)
         end
-        if next_packet_events.pet_status_change then
+        if next_packet_events.pet_status_change and not gearswap_disable then
```

Comment:
It's called `gearswap_disable` in this line, but `gearswap_disabled` in the other two changed lines. Is that intended?

Reply:
I wow, I didn't even see those two lines at all. Good eye, and sorry for my mix up.